spec/amp-actions-and-events.md

```diff
@@ -71,84 +71,123 @@ For example, the following is possible in AMP.
 ```
 
 ## Element Specific Events
+### form
 <table>
   <tr>
-    <th width="30%">Tag</th>
     <th>Event</th>
     <th>Description</th>
   </tr>
   <tr>
-    <td width="30%">form</td>
     <td>submit</td>
     <td>Fired when the form is submitted</td>
   </tr>
   <tr>
-    <td width="30%">form</td>
     <td>submit-success</td>
     <td>Fired when the form submission response is success.</td>
   </tr>
   <tr>
-    <td width="30%">form</td>
     <td>submit-error</td>
     <td>Fired when the form submission response is an error.</td>
   </tr>
 </table>
 
 
 ## Element Specific Actions
+### * (all elements)
 <table>
   <tr>
-    <th width="30%">Tag</th>
     <th>Action</th>
     <th>Description</th>
   </tr>
   <tr>
-    <td width="30%">*</td>
     <td>hide</td>
     <td>Hides the target element</td>
   </tr>
+</table>
+
+### amp-sidebar
+<table>
   <tr>
-    <td width="30%">amp-sidebar</td>
     <td>open (default)</td>
     <td>Opens the sidebar</td>
   </tr>
   <tr>
-    <td width="30%">amp-sidebar</td>
     <td>close</td>
     <td>Closes the sidebar</td>
   </tr>
   <tr>
-    <td width="30%">amp-sidebar</td>
     <td>toggle</td>
     <td>Toggles the state of the sidebar</td>
   </tr>
+</table>
+
+### amp-lightbox
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
   <tr>
-    <td width="30%">amp-lightbox</td>
     <td>open (default)</td>
     <td>Opens the lightbox</td>
   </tr>
   <tr>
-    <td width="30%">amp-lightbox</td>
     <td>close</td>
     <td>Closes the lightbox</td>
   </tr>
   <tr>
-    <td width="30%">amp-image-lightbox</td>
     <td>(default)</td>
```

Comment:
Shouldn't this row be specific to the `amp-image-lightbox` element?  Also, should the action be named as `open (default)` instead of `(default)`?